website/data/docs-nav-data.json

```diff
@@ -1498,98 +1498,98 @@
     "routes": [
       {
         "title": "Overview",
-        "path": "autoscaling"
+        "href": "/plugins/autoscaling"
```

Comment:
These should all be `/tools/autoscaling` 👀

Reply:
Thanks for catching that! - Fixed 39bc48a

Reply:
2 more stragglers - https://github.com/hashicorp/nomad/pull/11584/commits/705fa9fb006cde6a2bd02b1961e88fa10c846fe0